glusterfs/containers/0.2.1/glusterfs/Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM centos:7
 
-RUN yum install -y wget && \
-    wget http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-5.noarch.rpm && \
-    rpm -ivh epel-release-7-5.noarch.rpm && \
-    wget -P /etc/yum.repos.d http://download.gluster.org/pub/gluster/glusterfs/LATEST/EPEL.repo/glusterfs-epel.repo && \
-    yum install -y glusterfs-server glusterfs glusterfs-fuse jq curl && \
-    mv /var/lib/glusterd/hooks/1/set/post/S30samba-set.sh /var/lib/glusterd/hooks/1/set/post/dS30samba-set.sh && \
+RUN yum install -y wget
+RUN wget http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-6.noarch.rpm && \
+    rpm -ivh epel-release-7-6.noarch.rpm
+RUN wget -P /etc/yum.repos.d http://download.gluster.org/pub/gluster/glusterfs/LATEST/EPEL.repo/glusterfs-epel.repo && \
+    yum install -y glusterfs-server glusterfs glusterfs-fuse jq curl
+RUN mv /var/lib/glusterd/hooks/1/set/post/S30samba-set.sh /var/lib/glusterd/hooks/1/set/post/dS30samba-set.sh && \
```

Comment:
Multiple RUN lines create new layers in the image. I prefer to keep them joined into a single command.